ros/src/sensing/filters/packages/points_downsampler/nodes/voxel_grid_filter/voxel_grid_filter.cpp

```diff
@@ -5,7 +5,8 @@
  *  Redistribution and use in source and binary forms, with or without
  *  modification, are permitted provided that the following conditions are met:
  *
- *  * Redistributions of source code must retain the above copyright notice, this
+ *  * Redistributions of source code must retain the above copyright notice,
+ * this
```

Comment:
Line feeds should not be inserted into copyright statement. You might not use [roscpp_code_format](https://github.com/davetcoleman/roscpp_code_format). It is recommended to revert 3877f2c first, and then to use clang-format again.

[ROS CppStyleGuide](http://wiki.ros.org/CppStyleGuide) says 
```
6.1 Line length
Maximum line length is 120 characters.
```

and [roscpp_code_format](https://github.com/davetcoleman/roscpp_code_format) sets ColumnLimit 120.
```
ColumnLimit:    120
```